2_FirstComponent.md

```diff
@@ -146,24 +104,38 @@ As mentioned above, when the template containing the can.Component's tag is pars
 ![](images/2_first_component/ComponentTagRenderedHTML.png)
 
 ####Template
-The `template` property of the can.Component contains the string value of the can.Component's template. Note that the template property just contains a string value. You can inline the template, if it is small. However, the recommended way of working with templates, to maintain separation of concerns, is to keep them in their own files and load them using can.view, as we have done here.
+The `template` property of the `can.Component` contains the string value of the `can.Component`'s template. Note that the template property just contains a string value. You can inline the template, if it is small. However, the recommended way of working with templates, to maintain separation of concerns, is to keep them in their own files and load them using `can.view`, as we have done here.
```

Comment:
I'm not sure myself. I think what I was trying to say is that DocumentJS probably has a link/format for these things already.